renovate.json

```diff
@@ -3,12 +3,30 @@
   "extends": ["github>ministryofjustice/hmpps-renovate-config:jvm"],
   "assigneesFromCodeOwners": true,
   "prBodyTemplate": "{{{table}}}{{{notes}}}{{{warnings}}}{{{controls}}}",
+  "vulnerabilityAlerts": {"minimumReleaseAge": null},
   "packageRules": [
+    {
+      "matchManagers": ["gradle"],
+      "matchPackageNames": ["swagger-parser"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "swagger-parser",
+      "groupSlug": "swagger-parser",
+      "minimumReleaseAge": "3 days"
+    },
+    {
+      "matchManagers": ["gradle"],
+      "matchPackageNames": ["hmpps-digital-prison-reporting-lib"],
```

Comment:
do u wanna separate this in order to protect against non-major-version breaking changes? not against this at all, but that was much more of an issue in the ui app